addons/web/static/src/views/fields/field.js

```diff
@@ -80,6 +80,9 @@ export class Field extends Component {
             const fieldType = this.props.record.fields[this.props.name].type;
             this.FieldComponent = getFieldClassFromRegistry(fieldType, this.props.type);
         }
+        if (typeof this.props.showTooltip == "undefined") {
+            this.props.showTooltip = this.props.fieldInfo.noLabel;
+        }
```

Comment:
Generally speaking, it is always better to avoid writing on the props. The idea is to leave the props as the original information received from the parent.

In this case, a better fix will be : 

```suggestion
       this.showTooltip = this.props.showTooltip;
        if (typeof this.showTooltip == "undefined") {
            this.showTooltip = this.props.fieldInfo.noLabel;
        }
```

And modify in `get tooltip(...` to use `this.showTooltip` and not `this.props.showTooltip`.

But again, I'm not sure that this is the best fix, or if it needed to be fixed.